flytekit/configuration/plugin.py

```diff
@@ -52,6 +53,12 @@ def get_default_image() -> Optional[str]:
     def get_auth_success_html(endpoint: str) -> Optional[str]:
         """Get default success html for auth. Return None to use flytekit's default success html."""
 
+    @staticmethod
+    def get_additional_info_for_execution(
+        console_http_domain: str, entity: Union[FlyteWorkflowExecution, FlyteNodeExecution, FlyteTaskExecution]
```

Comment:
@ddl-rliu  What do you think of passing in `FlyteRemote` instead of `console_http_domain`? This will give the CLI plugin more context if they need it.

Reply:
Great, that makes sense to me! Done: https://github.com/flyteorg/flytekit/pull/2661/commits/c4231c2de6ca184ba67082fe87acbe3dfb8c9963

Reply:
@pingsutw (sorry for the ping) I think my commit here caused your approval to expire, could you take another look when you have the time? Thank you!